packages/flutter/lib/src/material/data_table.dart

```diff
@@ -205,6 +215,16 @@ class DataRow {
   ///    <https://material.io/design/interaction/states.html#anatomy>.
   final MaterialStateProperty<Color?>? color;
 
+  /// The cursor for a mouse pointer when it enters or is hovering over the
+  /// data row.
+  ///
+  /// If this is null, then the value of [DataTableThemeData.dataRowCursor]
+  /// is used. If that's null, then [MaterialStateMouseCursor.clickable] is used.
```

Comment:
It looks like the default is null. Does that always mean MaterialStateMouseCursor.clickable?

Reply:
Yep! `InkResponse` defaults to `MaterialStateMouseCursor.clickable`.

https://github.com/flutter/flutter/blob/250a6c294264d3e8b39b5580737f2ed959d1de5a/packages/flutter/lib/src/material/ink_well.dart#L395